src/main/java/com/ssbb/kanban/controller/project/ProjectController.java

```diff
@@ -26,9 +24,25 @@
 
 	@RequestMapping(value = "project", method = RequestMethod.GET)
 	public String loadProject(ModelMap map, HttpSession session) {
+
 		return "project";
 	}
 
+	@RequestMapping(value = "loadSelectedProject", method = RequestMethod.POST)
+	public String loadSelectedProject(ModelMap map, HttpSession session,
+			Project submittedProject) {
+
+		User user = (User) session.getAttribute(Constants.USER);
+		for (Project userProject : user.getProjects()) {

```

Comment:
@ilya123s I have added the above method in pull request #9 #.  Can you take a look at that and merge if you're happy?  Then we can test using that method.